maintenance/updateExternalLinks.php

```diff
@@ -77,6 +79,23 @@ public function execute() {
 
 		$this->output( "Script took {$time} seconds.\n" );
 	}
+
+	// Apparently, MediaWiki URL-encodes the whole URL, including the domain name,
```

Comment:
This should be a comment in the format of `/**` not `//` since it's a function comment.